design/system-snapshot.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Accumulo System Snapshot Design
+---
+
+## Disclaimer
+
+This is a work in progress design document for a feature that may never be
+implemented.  It represents one possible design for {% ghi 1044 %}, but does
+not have to be the only one.
+
+## Overview
+
+Being able to take snapshots of an entire Accumulo system and roll back to
+snapshot would support many administrative use cases.  This design outlines one
```

Comment:
could mention some use cases:
* nightly snapshots could be created to recover from major failures (and only lose the current days changes)
* snapshots could created before upgrades to enable rolling back if upgrade fails

Reply:
These are mentioned in the issue.  I will mention that.